app/models/user.rb

```diff
@@ -40,6 +42,8 @@ class User < ApplicationRecord
   scope :search_user, -> role, data {where("role = ? AND (phone_number = ? OR
     name LIKE ?)", role, data, "%#{data}%")}
   scope :is_online, -> {where online: true}
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/MultilineMethodCallIndentation
**Severity**: INFO
**File**: app/models/user.rb L44

Align `.new` with `Twilio::REST::Client` on line 43.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*